### PR TITLE
Translate `Binding.md` V4 zhCN (progress 30%)

### PR DESCRIPTION
**Pull request by likexoo**

### CODE REVIEW

**Comment by likexoo** [comment]
I have signed the CLA.

**Comment by dhmlau** [comment]
@likexoo, thanks for the PR. Could you please rebase your PR? It's out of date. Thanks.

**Comment by likexoo** [comment]
@dhmlau I fixed.

File: pages/zh/lb4/Binding.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+---
+lang: zh
+title: 'Binding'
+keywords: LoopBack 4.0, LoopBack 4, Node.js, TypeScript, OpenAPI
+sidebar: lb4_sidebar
```
Comment (dhmlau) [inline pages/zh/lb4/Binding.md]:
```suggestion
sidebar: zh_lb4_sidebar
```

File: pages/zh/lb4/Binding.md [hunk 2]
```diff
@@ -0,0 +1,192 @@
...
+permalink: /doc/zh/lb4/Binding.html
+---
```
Comment (dhmlau) [inline pages/zh/lb4/Binding.md:8]: add one more line after `permalink` line to indicate it's translation. See https://raw.githubusercontent.com/strongloop/loopback.io/gh-pages/pages/zh/lb4/Relations.md as an example. 

```
layout: translation
```

**Review by dhmlau — COMMENTED** [review]
@likexoo, just realized there's some changes needed for the front-matter. Please see my review comments. Thanks.

**Comment by likexoo** [comment]
@dhmlau Fixed and thanks for guiding

**Review by dhmlau — APPROVED** [review]

**Comment by dhmlau** [comment]
@likexoo, thanks for your translation PR. Your PR has landed! 🎉

**Pull request merged**